math/prime/millerrabinprimalitytest_test.go

```diff
@@ -36,6 +41,10 @@ func TestMillerRabinTest(t *testing.T) {
 
 func BenchmarkMillerRabinPrimalityTest(b *testing.B) {
 	for i := 0; i < b.N; i++ {
-		_, _ = MillerRabinTest(23, 5)
+		_, _ = MillerRabinProbabilistic(23, 5)
+	}
+
+	for i := 0; i < b.N; i++ {
```

Comment:
Separate out benchmarks please - they skew the actual results.

Reply:
Done.